chat/src/main/java/bisq/chat/notifications/ChatNotificationService.java

```diff
@@ -367,10 +367,13 @@ private <M extends ChatMessage> ChatNotification createNotification(String id, C
                         .map(service -> service.getChannelTitle(chatChannel))
                         .orElse(Res.get("data.na"));
                 channelInfo = StringUtils.createNavigationPath(
-                        chatChannel.getChatChannelDomain().getDisplayString(), channelTitle);
+                        " - ",
```

Comment:
Do we need to customize that delimiter?

Reply:
I added this because I didn't like how the hyphen looked like, therefore the option to change it.
I went ahead and changed it to `>` in #1561. I think the latter is more appropriate for this context, and it looks better in general and in particular in cases where `-` is already in the channel title like `Off-topic`.